Source/textdat.cpp

```diff
@@ -645,6 +645,12 @@ const TextDataStruct alltext[] = {
 	{ "In Spiritu Sanctum. |", 1, 5, PS_ROGUE54 },
 	{ "Praedictum Otium. |", 1, 5, PS_ROGUE55 },
 	{ "Efficio Obitus Ut Inimicus. |", 1, 5, PS_ROGUE56 },
+	{ " |", 0, 5, TSFX_COW3 },
+	{ " |", 0, 5, TSFX_COW4 },
+	{ " |", 0, 5, TSFX_COW5 },
+	{ " |", 0, 5, TSFX_COW6 },
+	{ " |", 0, 5, TSFX_COW7 },
+	{ " |", 0, 5, TSFX_COW8 },

```

Comment:
Text is missing

Reply:
@AJenbo I left empty on purpose (following TSFX_COW1 and TSFX_COW2). Those sounds seem like real farm animals recordings. :sweat_smile: